loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -36,6 +36,10 @@ public enum VoltageInitMode {
     public enum BalanceType {
         PROPORTIONAL_TO_GENERATION_P,
         PROPORTIONAL_TO_GENERATION_P_MAX,
+        PROPORTIONAL_TO_GENERATION_REMAINING_MARGIN,
```

Comment:
Remaining margin means proportional to pmax - targetP ?

Reply:
PROPORTIONAL_TO_GENERATION_REMAINING_MARGIN seems ok to me

Reply:
Maybe we should you add a comment to describe exactly what does it mean

Reply:
I added javadoc to the options